cmd/delete.go

```diff
@@ -26,8 +26,8 @@ var componentDeleteCmd = &cobra.Command{
 	`,
 	Args: cobra.MaximumNArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
-		log.Debugf("component delete called")
-		log.Debugf("args: %#v", strings.Join(args, " "))
+		glog.V(4).Infof("component delete called")
```

Comment:
Should we document this at all anywhere for future devs?

Reply:
Otherwise, this LGTM.

Reply:
Done raised https://github.com/redhat-developer/odo/pull/673